packages/types/src/package.ts

```diff
@@ -2,4 +2,6 @@
 export interface Package {
```

Comment:
m: Would it make sense to type every field here as optional instead of using Partial in other places?

Reply:
This type is already used and in the @sentry/types package. Hence, making name and version optional would be breaking. I was debating to just create another package.json type in the SvelteKit SDK package but I'd like to avoid the duplication here. 
The only reason why this is`Partial` though is the early return. We can change the function to get rid of it.